src/posts/2022-02-10-design-het-web.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Design het Web'
+category: inspiration
+tags: new blogpost
+comments: true
+author: joris
+min_read: 5
+date: 2022-02-10
+permalink: /2022/02/10/design-het-web/
+---
+
+<!-- Section: Intro -->
+
+Development en webdesign zijn termen die vaak door elkaar worden gehaald. Hoewel het twee aparte taken zijn varen ze wel in hetzelfde vaarwater en zullen ze in een groot deel van de web-trajecten overlappen. Uit mijn ervaring blijkt dat veel personen die in één van deze twee categorieën werken ook behendig zijn in de ander, maar hebben altijd voorkeur voor hun eigen onderdeel. Welke mogelijkheden zijn er voor wie als we het hebben over doorgroeien(als designer)?
+
+Er zijn een aantal mogelijkheden waar je uit kan kiezen als designer: je wil doorleren en de code echt gaan beheersen, je wilt omscholen en het design achter je laten, je vindt allebei leuk en blijft er tussenin hangen of je sluit de code helemaal uit.
+
+## Vergeet het design
+
+Er is een mogelijkheid om het designen helemaal te laten voor wat het is en je vol te storten op de code. Dit is iets wat ik niet zou aanraden; veel bedrijven zijn blij met developers die ook designkennis hebben. Om binnen organisaties de overlap tussen design en development zo soepel mogelijk te laten lopen zijn een aantal, of tenminste één zo’n persoon nodig. Je kunt dus een belangrijke rol gaan spelen.
+
+In het begin heb je nog niet veel aan je design skills, dit komt omdat je nog heel beperkt bent in wat je kunt maken. In een later stadium zul je gaan merken dat je niet alleen vette dingen kunt maken, maar ook verzinnen. Je kunt dan nauw samenwerken met de designers, en ze zullen ook blij zijn met jou.
+
+## Leer het bij: Code, Code, Code!
+
+In het begin is het niet te overzien; verschillende tools, talen & frameworks; Vue, Angular, React etc!
+Ook worden er veel fancy Engelse namen en code talen gebruikt in dagelijks communicatie. Afhankelijk van je doel is de learning curve best stijl, zeker als je alleen een design achtergrond hebt. Er zijn ook veel overeenkomsten, die zie je vooral terug in HTML/CSS. Denk aan eigenschappen van elementen, werkstructuur en het uiteindelijke doel; gave dingen maken!
```

Comment:
> code talen

What would this refer to?
Or are we talking 'programmeertalen'?

>de learning curve best stijl

_learning curve_ -> _leercurve_?
_stijl_ -> _steil_?